packages/plugin-ext/src/main/browser/view/plugin-view-widget.ts

```diff
@@ -96,10 +100,39 @@ export class PluginViewWidget extends Panel implements StatefulWidget {
         }
     }
 
+    private _message: string | undefined;
+    get message(): string | undefined {
+        return this._message;
+    }
+
+    set message(message: string | undefined) {
+        this._message = message;
+        this.updateWidgetMessage();
+    }
+
+    private updateWidgetMessage(): void {
+        const widget = this.widgets[0];
+        if (widget) {
+            if (widget instanceof TreeViewWidget) {
```

Comment:
we should set `message` directly to `TreeViewWidget`, `PluginVIewWidget` does not need to deal with it

Reply:
Again, `TreeViewWidget` is not accessible at the moment of `$setMessage` call, this is why I save the message in `PluginVIewWidget` untill `addWidget` or `insertWidget` is invoked.